libnavigation-core/src/main/java/com/mapbox/navigation/core/formatter/MapboxDistanceUtil.kt

```diff
@@ -16,6 +16,8 @@ import kotlin.math.roundToInt
  */
 object MapboxDistanceUtil {
 
+    private const val smallDistanceLowerBoundInMiles = 0.1
```

Comment:
constants must be `UPPER_CASE`

Reply:
Why on Earth doesn't ktlint check that? :)

Reply:
Unfortunately, this rule does not exist https://pinterest.github.io/ktlint/rules/standard/ 😄 adding and testing a custom `rule` for this will take time. it's simply not worth it

Reply:
I think detekt knows about this rule. Because I've definitely senn this check somewhere and I think it was detekt.

Reply:
It was disabled some time ago at https://github.com/detekt/detekt/pull/3151 because of `Jetpack Compose` [contract](https://github.com/androidx/androidx/blob/androidx-main/compose/docs/compose-api-guidelines.md#singletons-constants-sealed-class-and-enum-class-values).